nemo/collections/asr/models/ctc_models.py

```diff
@@ -663,6 +705,72 @@ def test_dataloader(self):
         if self._test_dl is not None:
             return self._test_dl
 
+    def _setup_pseudo_label_dataloader(self, cache_manifest: str, audio_tar: str = None, batch_size: int = 64):
```

Comment:
is this not required for RNNTModels?

Reply:
We provide SlimIPL only for CTC and Hybrid models.